charms/jupyter-ui/terraform/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "app_name" {
+  value = juju_application.jupyter_ui.name
+}
+
+output "provides" {
```

Comment:
What's the best practice for when the charm does not have provide/require or both? I couldn't find it in the CC006 spec, so I wonder if it's enough by just removing it.

Reply:
We 've discussed with @mvlassis and thought that it's better to output an empty value to keep a consistent API and expose that there are no endpoints. WDYT about this?

Reply:
Sounds good for consistency.